extensions/WEBGL_polygon_mode/extension.xml

```diff
@@ -19,6 +19,13 @@
     <mirrors href="https://chromium.googlesource.com/angle/angle/+/HEAD/extensions/ANGLE_polygon_mode.txt"
              name="ANGLE_polygon_mode">
     </mirrors>
+
+    <p>
+      <b>Note:</b> Browsers should should discourage use of this extension in non-debug desktop
+      applications due to its non-portability (it is generally only supported on desktop platforms,
```

Comment:
Saying "desktop" here is a bit misleading because the extension is supported on iOS/iPadOS.

Reply:
Good point, rewrote the portability warning. How does this look?